examples/webgl_geometry_compression.html

```diff
@@ -0,0 +1,298 @@
+			folder.add(data, 'oct2bytesEncode', false).onChange(generateGeometry);
+
+			folder = gui.addFolder('UV Compression');
+			folder.open();
+			folder.add(data, 'defaultEncodeUV', false).onChange(generateGeometry);
+
+			folder = gui.addFolder('Memory Info');
+			folder.open();
+			memoryDisplay = folder.add(data, 'totalGPUMemory', "0 bytes");
+			computeGPUMemory(ballMesh);
+
+			//
+
+			if (statsEnabled) {
+
+				stats = new Stats();
+				container.appendChild(stats.dom);
+
+			}
+
+			window.addEventListener('resize', onWindowResize, false);
+
+		}
+
+		//
+
+		function onWindowResize() {
+
+			renderer.setSize(window.innerWidth, window.innerHeight);
+
+			camera.aspect = window.innerWidth / window.innerHeight;
+			camera.updateProjectionMatrix();
+
+		}
+
+		//
+		function updateLightsPossition() {
+			lights.forEach(light => {
+				var direction = light.position.clone();
+				direction.applyAxisAngle(new THREE.Vector3(1, 1, 0), data.rotationSpeed / 180 * Math.PI);
+				light.position.add(direction.sub(light.position));
+			});
+		}
+
+		//
+
+		function animate() {
+
+			requestAnimationFrame(animate);
+
+			controls.update();
+			updateLightsPossition();
+
+			renderer.render(scene, camera);
+
+			if (statsEnabled) stats.update();
+
+		}
+
+		//
+
+		function updateGroupGeometry(mesh, lineSegments, geometry, data) {
+
+			if (geometry.isGeometry) {
+
+				geometry = new THREE.BufferGeometry().fromGeometry(geometry);
+
+				console.warn('THREE.GeometryBrowser: Converted Geometry to BufferGeometry.');
+
+			}
+
+			lineSegments.geometry.dispose();
+			mesh.geometry.dispose();
+
+			lineSegments.geometry = new THREE.WireframeGeometry(geometry);
+			mesh.geometry = geometry;
+			mesh.material = new THREE.MeshPhongMaterial({ color: 0xffffff, emissive: 0x111111 });
+			mesh.material.map = data.texture ? texture : null;
+
+			var normalEncode = "";
+			if (data.oct1bytesEncode) {
+				/**
+				 * It is not recommended to use 1-byte octahedron normals encoding unless you want to extremely reduce the memory usage
+				 * As it makes vertex data not aligned to a 4 byte boundary which may harm some WebGL implementations and sometimes the normal distortion is visible
+				 * Please refer to @zeux 's comments in https://github.com/mrdoob/three.js/pull/18208
+				 */
+				normalEncode = "OCT1Byte";
+
+			} else if (data.oct2bytesEncode) {
+
+				normalEncode = "OCT2Byte";
+
+			} else if (data.anglesEncodeNormal) {
+
+				normalEncode = "ANGLES";
+
+			} else if (data.defaultEncodeNormal) {
+
+				normalEncode = "DEFAULT";
+
+			}
+
+			if (normalEncode != "") {
+				GeometryCompressionUtils.compressNormals(mesh, normalEncode);
+			}
+
+			if (data.quantizeEncodePos) {
+				GeometryCompressionUtils.compressPositions(mesh);
+			}
+
+			if (data.defaultEncodeUV) {
+				GeometryCompressionUtils.compressUvs(mesh);
+			}
+
+			computeGPUMemory(mesh);
+
+		}
+
+
+		function computeGPUMemory(mesh) {
```

Comment:
There's already an `estimateBytesUsed` function in [BufferGeometryUtils](https://github.com/mrdoob/three.js/blob/dev/examples/jsm/utils/BufferGeometryUtils.js#L457). It might be preferred to use that one for the sake of consistency?